src/Core/src/Platform/Windows/WindowExtensions.cs

```diff
@@ -9,21 +10,31 @@ public static void UpdateTitle(this UI.Xaml.Window nativeWindow, IWindow window)
 			nativeWindow.Title = window.Title;
 
 			var rootManager = window.Handler?.MauiContext?.GetNavigationRootManager();
-			if(rootManager != null)
+			if (rootManager != null)
 			{
 				rootManager.SetWindowTitle(window.Title);
 			}
 		}
 
 		public static IWindow? GetWindow(this UI.Xaml.Window nativeWindow)
 		{
-			foreach(var window in MauiWinUIApplication.Current.Application.Windows)
+			foreach (var window in MauiWinUIApplication.Current.Application.Windows)
 			{
 				if (window?.Handler?.NativeView is UI.Xaml.Window win && win == nativeWindow)
 					return window;
 			}
 
 			return null;
 		}
+
+		public static IntPtr GetWindowHandle(this UI.Xaml.Window nativeWindow)
+		{
+			var hwnd = nativeWindow.As<IWindowNative>().WindowHandle;
```

Comment:
You can use the new WinRT Interop helpers and remove the IWindowNative definition:
```cs
 IntPtr hwnd = WinRT.Interop.WindowNative.GetWindowHandle(m_window);
```

Reply:
Thanks! Updated to use that and deleted a file from our code.